keras_nlp/utils/text_generation_test.py

```diff
@@ -342,7 +406,7 @@ def test_generate_with_ragged_prompt(self):
     def test_assert_probability_distribution_generation_is_correct(self):
         def token_probability_fn(inputs):
             batch_size = inputs.shape[0]
-            prob = tf.constant([[0.01, 0.01, 0.08, 0.9]])
+            prob = tf.constant([[0.0, 0.0, 0.0, 1.0]])
```

Comment:
Do we need to change the number here? The original value seems to be more general?

Reply:
Ah, yes. This was done to take care of accelerator testing. Seeded generation does not work, so, we've made the probability 1.